### DISCUSSION TITLE
release/v2

### DISCUSSION THREAD

**Discussion by giscus** [gogf/gf-site#58] [General]
# release/v2.8

此版本更新包含多个重要改进，包括最低Golang版本要求从1.18调整为1.20，废弃gring组件，删除jaeger组件，改进gjson参数形式，以及提高ghttp和gudp的灵活性。新特性包括数据库连接支持unix socket，支持time字段类型，gdb增加Exist方法，提升gconv转换性能，改进gvalid校验规则与gtest断言方法，增强gcron定时任务管理，并显著提升GoFrame开发工具的功能和扩展性。

https://goframe.org/release/v2.8

**Comment by hailaz** [comment]
666

**Comment by sanrentai** [comment]
drivers/mssql 分页生成的sql语句不支持 sql server 2008r2

**Reply by gqcn** [reply]
在`v2.8.3`版本中将会支持。
